docs/configuration.md

```diff
@@ -0,0 +1,457 @@
+>>##### Description: Defined default audio track language.
+>>If audio track with the defined language is exists, this audio track will be selected as the initial audio track.
+>> #### Example:
+>>```js
+>>var config = {
+>>    playback: {
+>>        audioLanguage: "eng" // Start playback with english audio
+>>    }
+>>};
+>>```
+>## 
+>>### config.playback.textLanguage
+>>##### Type: `string || "auto"`
+>>##### Default: `""`
+>>##### Description: Defined default captions language.
+>>If captions with the defined language is exists, this text track will be selected as the initial text track.
+>> #### Example:
+>>```js
+>>var config = {
+>>    playback: {
+>>        textLanguage: "heb" // Start playback with hebrew captions
+>>    }
+>>};
+>>```
+>> If the value `"auto"` is set, i.e:
+>>```js
+>>var config = {
+>>    playback: {
+>>        textLanguage: "auto"
+>>    }
+>>};
+>>```
+>> The player will choose default captions language by the following logic:
+>> 1. **Locale language** - If there are captions in the user's system language then this language will be selected.
+>> 2. **Manifest default language** - If a default language is specified in the manifest file then this language will be selected.
+>> 3. **First language in manifest** - The first language specified in the manifest file will be selected.
+>> 4. If none of the above conditions have taken place, do not display captions.
+>## 
+>>### config.playback.useNativeTextTrack
+>>##### Type: `boolean`
+>>##### Default: `false`
+>>##### Description: Whether to use native browser text tracks.
+>>If sets to true, native browser captions will be displayed. 
+>## 
+>>### config.playback.volume
+>>##### Type: `number`
+>>##### Default: `1`
+>>##### Description: Defined the initial volume value.
+>>Value must be in the range of 0-1.
+>> #### Example:
+>>```js
+>>var config = {
+>>    playback: {
+>>        volume: 0.5
+>>    }
+>>};
+>>```
+>## 
+>>### config.playback.playsinline
+>>##### Type: `boolean`
+>>##### Default: `true`
+>>##### Description: A Boolean attribute which indicates that the video is to be played "inline", that is within the element's playback area.
+>>Especially relevant for playing on iPhone devices, where if the value is set to false, the video will be played using the AV Player (iOS native video player).
+>## 
+>>### config.playback.preload
+>>##### Type: `string`
+>>##### Default: `"none"`
+>>##### Description: Indicates whether the video should be preloaded or not.
+>>Possible values:
+>> - `"none"`: indicates that the video should not be preloaded.
+>> - `"auto"`: indicates that the whole video file could be downloaded, even if the user is not expected to use it.
+>##
+>>### config.playback.autoplay/allowMutedAutoPlay
+>>for `autoplay` & `allowMutedAutoPlay` options read [here](autoplay.md).
+>## 
+>>### config.playback.muted
+>>##### Type: `boolean`
+>>##### Default: `false`
+>>##### Description: Indicates whether the video should be muted or not.
```

Comment:
**Description: Indicates whether the video should be muted or not. This is a Boolean attribute that indicates the default setting of the audio contained in the video. If set, the audio will be initially silenced. The attribute's default value is false, which means that the audio will be played automatically when the video is played.**